README.md

```diff
@@ -30,3 +31,32 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+Recurrence relation for mystery():
+
+$T(n) =$ 1 if $n <= 1$ <br>
+$3T(\frac{n}{3}) + n^5$ otherwise <br>
+
+Solving by substitution:<br>
+$T(n) = 3T(\frac{n}{3}) + n^5$<br>
+$T(n) = 3(3T(\frac{n/3}{3}) + (\frac{n}{3})^5) + n^5$<br>
+$T(n) = 3^2T(\frac{n}{3^2}) + \frac{n^5}{3^4} + n^5$<br>
+Skipping the substitution step the next experession is:<br>
+$T(n) = 3^3T(\frac{n}{3^3}) + \frac{n^5}{3^5} + \frac{n^5}{3^4} + n^5$<br>
+
+Analysing the pattern, the expression for this recurrence relation is: $T(n) = 3^iT(\frac{n}{3^i}) + \sum{i=2}^k \frac{n^5}{3^{i+2}}  + n^5$<br>
```

Comment:
I think you're missing a _ in your notation for the sum. And use $j$ or another variable (unless you really mean that it's the same $i$?).

Reply:
Not sure what you mean. Where is the underscore supposed to go?
I do mean its the same as i, as the sum starts when i = 2. Should I use a different variable anyway? Thanks!!

Reply:
Notation looks ok now, but I don't understand what you're summing over. This can't be $i$ (in the same way that when you have two nested loops you can't have the same loop variable in each). And what is $k$?

Reply:
Hm okay. I'm summing over every \frac{n^5}{3^{i+2}} up to a certain number, and I guess that number would be the number of recurrences, so that's what $k$ is supposed to represent. I suppose $k = n$ would apply, since n would be the number of recurrences. I updated the variables, hopefully its on the right track now.